godtools/GeneralExtensions/UIFont.swift

```diff
@@ -0,0 +1,57 @@
+//
+//  UIFont.swift
+//  godtools
+//
+//  Created by Pablo Marti on 7/18/17.
+//  Copyright © 2017 Cru. All rights reserved.
+//
+
+import UIKit
+
+extension UIFont {
+    
+    static func gtRegular(size: CGFloat) -> UIFont {
+        return UIFont.systemFont(ofSize: size, weight: UIFontWeightRegular)
+    }
+    
+    static func gtLight(size: CGFloat) -> UIFont {
+        return UIFont.systemFont(ofSize: size, weight: UIFontWeightLight)
+    }
+    
+    static func gtThin(size: CGFloat) -> UIFont {
+        return UIFont.systemFont(ofSize: size, weight: UIFontWeightThin)
+    }
+    
+    static func gtSemiBold(size: CGFloat) -> UIFont {
+        return UIFont.systemFont(ofSize: size, weight: UIFontWeightSemibold)
+    }
+    
+    func transformToAppropriateFont() -> UIFont {
```

Comment:
Can you make this function private and call this function inside the functions above instead of throughout the various tract classes? If that works, it's better because we will not have to repeat this function call whenever a new UI element is added.